### PR TITLE
Fix event property null value crash

### PR DESCRIPTION
**Pull request by olmisa**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Track event crash caused by event property null value.

## Related PRs or issues

[AB#81360](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/81360)
https://github.com/microsoft/appcenter-sdk-unity/issues/456/

### CODE REVIEW

**Review by annakocheshkova — DISMISSED** [review]

File: Assets/AppCenter/Plugins/iOS/Core/Utility/NSStringDictionaryHelper.mm [hunk 1]
```diff
@@ -27,7 +27,7 @@
   NSMutableDictionary<NSString *, NSString *> *nsdictionary = [[NSMutableDictionary alloc] initWithCapacity:count];
   for (int i = 0; i < count; ++i) {
     NSString *key = [NSString stringWithUTF8String:keys[i]];
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Core/Utility/NSStringDictionaryHelper.mm]: Let's use our convert method for the keys too

Comment (olmisa) [reply Assets/AppCenter/Plugins/iOS/Core/Utility/NSStringDictionaryHelper.mm]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # App Center SDK for Unity Change Log
 
+## Release 3.2.2 (in development)
+
+### App Center Analytics
+
+#### iOS
+
+* **[Fix]** Fix track event crash caused by event property null value.
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Fix `Analytics.TrackEvent` crash if event properties contain `null` value.
```

**Review by Jamminroot — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Review by annakocheshkova — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

**Comment by maximkornilov** [comment]
Guys, what are the status? Don't you think it looks weird that it takes about a month to fix a quite simple case which leads to **application crash**?

**Comment by olmisa** [comment]
@maximkornilov the issue was already fixed about a month, but CI fails due to Unity license problems. The fix will be merged after the Unity license problem solved.

**Comment by olmisa** [comment]
@maximkornilov the fix was already implemented about a month ago. But there is a problem with CI. The fix will be merged right after the CI problems will be solved.

**Comment by iamclement** [comment]
Closing/reopening again, hopefully the CI should work now.

**Pull request merged**